index.html

```diff
@@ -35,9 +35,290 @@ <h3>I am a software developer</h3>
+                                    <p class="sub">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                                    <section class="tags">
+                                        <ul class="tags1">
+                                            <li>HTML</li>
+                                        </ul>
+                                        <ul class="tags2">
+                                            <li>BOOSTSTRAP</li>
+                                        </ul>
+                                        <ul class="tags3">
+                                            <li>RUBY</li>
+                                        </ul>
+                                    </section>
+                                </section>
+
+                            </section>
+                        </section>
+                        <!-- <section class="mask-sub"></section> -->
+                    </section>
+
+                    <section class="action">
+                        <section class="but">
+                            <p class="see1">See Project</p>
+                        </section>
+                    </section>
+                </section>
+                <section class="card-works">
+                    <section class="mask">
+                        <section class="img-placeholder">
+                            <section class="div">
+                                <section class="right-works">
+                                    <p class="pro">Profesional Art Printing Data</p>
+                                    <p class="sub">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                                    <section class="tags">
+                                        <ul class="tags1">
+                                            <li>HTML</li>
+                                        </ul>
+                                        <ul class="tags2">
+                                            <li>BOOSTSTRAP</li>
+                                        </ul>
+                                        <ul class="tags3">
+                                            <li>RUBY</li>
+                                        </ul>
+                                    </section>
+                                </section>
+
+                            </section>
+                        </section>
+                        <!-- <section class="mask-sub"></section> -->
+                    </section>
+
+                    <section class="action">
+                        <section class="but">
+                            <p class="see1">See Project</p>
+                        </section>
+                    </section>
+                </section>
+            </section>
+            <section class="about">
```

Comment:
- [ ] Kindly add spacings in the about section to match the design template.
### Your design
![5](https://user-images.githubusercontent.com/94551484/192065204-66b793b3-fbca-4bdd-90b5-d724d57d73bf.png)
### Figma template
![f5](https://user-images.githubusercontent.com/94551484/192065224-e14d5ccd-1d2e-4659-b6d2-2470093689c2.png)